aws/mofo-projects/health-checks/health-checks.tf

```diff
@@ -0,0 +1,90 @@
+# Paging alerts are sent to victorops
+data "aws_sns_topic" "victorops-integration" {
```

Comment:
Are these SNS topics not managed by this config?

Reply:
No, I'm just getting the info. If at some point you want to control them with terraform, it's probably a good idea to create a "main shared resources" project.